### DISCUSSION TITLE
Support -o parameter via `next build` + `output: "export"` as `next export` did

### DISCUSSION THREAD

**Discussion by pascavi** [vercel/next.js#62612] [Ideas]
### Link to the code that reproduces this issue

https://github.com

### To Reproduce

* Run `nextjs build` with the `export` action on `next.config`. Add a `-o my-folder` parameter to override the export folder
* Folder is not overriden

### Current vs. Expected behavior

You can use the `-o` parameter in the same way that you could with the `export` command

### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
const nextConfig = {
  output: 'export'
}
```

### Which area(s) are affected? (Select all that apply)

CLI (create-next-app)

### Additional context

Being able to override the output folder was essential in our pipeline configuration.

**Comment by matdave** [comment]
@pascavi did you ever find a solution to this. I'm finally upgrading and ran into this issue as well.
